examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -104,7 +104,7 @@ ${ flowData.code }
 
 			} else if ( itemSize === 3 ) {
 
-				format = 6407; // patch since legacy doesn't use RGBFormat for rendering but here it's needed for packing optimization
+				format = RGBFormat;
```

Comment:
@RenaudRohlinger I believe you have originally added this code so I want to ping you to ensure nothing breaks with this change^^.

Reply:
Ah yes! I added the numeric value since we removed RGBFormat from the codebase in the past. I thought about using the numeric value of `gl.RGB` format here but indeed the logic you added back is the correct one 👍